zellij-tile/src/shim.rs

```diff
@@ -37,6 +37,10 @@ pub fn open_file(path: &Path) {
 pub fn set_timeout(secs: f64) {
     unsafe { host_set_timeout(secs) };
 }
+pub fn exec_cmd(cmd: &[String]) {
```

Comment:
Additionally, I might change this to take either `&[&str]` or maybe `&[impl AsRef<str> + Serialize]` if you really want to pass strings in directly.

Both solutions allow very simple calls like: 
```rs
exec_cmd(&["playerctl", "pause"]);
```

And the more complex type allows this as well, which you can decide whether you'll need or not:
```rs
let cmds: Vec<String> = vec!["playerctl".into(), "play".into()];
exec_cmd(&cmds);
```

Reply:
I prefer `&[&str]` because it's more friendly to the plugin as most of the time `&str` is sufficient.